ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/DecoderOptions.java

```diff
@@ -38,6 +38,13 @@
 
   void setGeoCityFilter(ValueProvider<String> value);
 
+  @Description("Path (local or gs://) to a .tar.gz file containing json schemas; the expected"
+      + " format is the output of GitHub's archive endpoint for mozilla-pipeline-schemas:"
+      + " https://github.com/mozilla-services/mozilla-pipeline-schemas/archive/dev.tar.gz")
```

Comment:
This will be handled by the ops infra more or less analogously to how it's handled now i.e. the schemas will be recompiled before the ops artifact is generated (we should do this follow-up at any rate of course).